src/main/java/org/testcontainers/containers/TarantoolContainer.java

```diff
@@ -74,38 +66,14 @@ public TarantoolContainer(String dockerImageName) {
         clientHelper = new TarantoolContainerClientHelper(this);
     }
 
-    /**
-     * Constructor for {@link TarantoolContainer}
-     *
-     * @param dockerImageName docker image name for container creating
-     * @param clientBuilder   client builder with custom client settings for setting up container
-     */
-    public TarantoolContainer(String dockerImageName,
-                              TarantoolClientBuilder clientBuilder) {
-        super(dockerImageName);
-        clientHelper = new TarantoolContainerClientHelper(this, clientBuilder);
-    }
-
     /**
      * Constructor for {@link TarantoolContainer}
      *
      * @param tarantoolImageParams params for cached image creating
      */
     public TarantoolContainer(TarantoolImageParams tarantoolImageParams) {
         super(TarantoolContainerImageHelper.getImage(tarantoolImageParams));
-        clientHelper = new TarantoolContainerClientHelper(this);
-    }
-
-    /**
-     * Constructor for {@link TarantoolContainer}
-     *
-     * @param tarantoolImageParams params for cached image creating
-     * @param clientBuilder        client builder with custom client settings for setting up container
-     */
-    public TarantoolContainer(TarantoolImageParams tarantoolImageParams,
-                              TarantoolClientBuilder clientBuilder) {
-        super(TarantoolContainerImageHelper.getImage(tarantoolImageParams));
-        clientHelper = new TarantoolContainerClientHelper(this, clientBuilder);
+        clientHelper = new TarantoolContainerClientHelper(this, "/sdk/tarantoolctl");
```

Comment:
This should be a constant at least, but better that should be either an env variable or the executable should be in the classpath so no explicit path is specified. DO not leave such hardcode, that will not work for all environments